src/software/thunderscope/replay/proto_player.py

```diff
@@ -119,10 +173,21 @@ def load_replay_chunk(replay_chunk_path: str) -> list:
                     line = log_file.readline()
                     if not line:
                         break
-                    cached_data.append(line)
+
+                    if not ProtoPlayer.is_log_entry_corrupt(line):
+                        cached_data.append(line)
+                    else:
+                        logging.warning(
+                            "There are log entries that are corrupted. Entries ignored!"
+                        )
                 except EOFError:
                     break
 
+                except Exception:
+                    logging.warning(
+                        "Some unknown exception have occured. Error ignored in ProtoPlayer"
+                    )
```

Comment:
lets make this more descriptive:
```suggestion
                except Exception as e:
                    logging.warning(
                        f"An unknown exception has occurred while reading {replay_chunk_path}: {e}"
                    )
```